docs/how-to-build-a-lexical-model.md

```diff
@@ -0,0 +1,365 @@
+is:
+
+    example.str.tutorial
+
+
+Step 4: Creating the package files
+----------------------------------
+
+Now that we have a unique ID for our model, and wordlist data, we can
+the appropriate packaging files.
+
+Say we're currently in a folder called `release`: our model will follow
+this template for its directory structure.
+
+```
+release/
+└── {author}/
+    └── {bcp47}.{uniq}/
+        ├── {author}.{bcp47}.{uniq}.model_info
+        └── source/
+            ├── {author}.{bcp47}.{uniq}.model.kps
+            ├── wordlist.tsv
+            └── model.ts
+```
+
+Notice how under `release`, there is a folder that is named after the
+author. Within that folder is a folder for each models created by that
+particular author. Within each model folder, this is a folder called
+`source` that contains **wordlist.tsv**. We will discuss what the other
+files are called a bit later in this document.
+
+For our example of `example.str.tutorial`, I'll want the following
+folder structure:
+
+```
+release/
+└── example/
+    └── str.tutorial/
+        ├── example.str.tutorial.model_info
+        └── source/
+            ├── example.str.tutorial.model.kps
+            ├── wordlist.tsv
+            └── model.ts
+```
+
+To create this folder structure, I can execute the following in
+Git Bash or the terminal:
+
+    cd /type/the/path/to/release
+    mkdir -p example/str.tutorial/source
+
+Now move the wordlist to the appropriate place:
+
+    mv /type/the/current/path/to/wordlist.tsv example/str.tutorial/source/
+
+I now have the following folder structure:
+
+```
+.
+└── example
+    └── str.tutorial
+        └── source
+            └── wordlist.tsv
+```
+
+We need to create the remaining files:
+ - The "model info" file: `example.str.tutorial.model_info`
+ - The package file: `example.str.tutorial.model.kps`
+ - The model configuration file: `model.ts`
+
+
+Step 4.1: Creating the model info file
+--------------------------------------
+
+The model info contains a little bit of metadata that is required when
+distributing packages. It is a JSON file that lists the model license
+and a short description of the lexical model.
+
+The `.model_info` file should follow this template:
+
+```json
+{
+    "license": "<SPDX license identifier>",
+    "description": "<short description of the lexical model>"
+}
+```
+
+Copy this template, and use it to create your own `.model_info` file.
+
+Save this file to `{author}/{bcp47}.{uniq}/{author}.{bcp47}.{uniq}.model_info`.
+
+> NOTE: the only current valid value for `license` is `"mit"`.
+
+In our `example.str.tutorial` example, I'll use the following
+`.model_info`.
+
+```json
+{
+    "license": "mit",
+    "description": "Predicts words in the SENĆOŦEN language (Saanich Dialect)"
+}
+```
+
+I'll save this file to `example/str.tutorial/example.str.tutorial.model_info`.
+
+
+Step 4.2: Creating the KPS file
+-------------------------------
+
+**TODO:**
```

Comment:
We'll generally encourage model developers to use Keyman Developer to create this file. It saves on silly errors. Clean support for models in Developer however is of course TBD.